.github/workflows/datafuse-perf-docker.yml

```diff
@@ -0,0 +1,35 @@
+name: Push to Dockerhub
+on:
+  push:
+    branches: master
```

Comment:
Only consider building when the `*.rs`/`*.toml` file is changed?

Reply:
updated path dependencies